tests/TestHelpers/GraphHelper.php

```diff
@@ -1983,4 +1983,31 @@ public static function enableShareSync(
 			\json_encode($body)
 		);
 	}
+
+	/**
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string $user
+	 * @param string $password
+	 * @param string $driveId
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public static function getDrivePermissionsUsingRootEndpoint(
```

Comment:
if we can make it similar to method `getPermissionsList()`
```suggestion
	public static function getDrivePermissionsList(
```

Reply:
it is better we specify root endpoint to avoid confusion for future maintainer

Reply:
we dont have another specific method for listing drive permissions. so renaming should be fine